src/tests/toml.rs

```diff
@@ -35,16 +37,40 @@ fn toml_check() -> Result<(), Box<dyn std::error::Error>> {
         let mut contents = String::new();
         file.read_to_string(&mut contents).unwrap();
         for matches in TOML_REGEX.captures_iter(&contents) {
-            let fence = matches.get(1).unwrap();
+            let cargo = {
+                let t = matches.get(1).unwrap().as_str();
+                if t.is_empty() {
+                    false
+                } else if t == ",cargo" {
+                    true
+                } else {
+                    println!("skipping {t}");
+                    continue;
+                }
+            };
+            let fence = matches.get(2).unwrap();
+            let fence_content = fence
+                .as_str()
+                .replace("$TARGET", "x86_64-unknown-linux-gnu")
+                .replace("${target}", "x86_64-unknown-linux-gnu");
```

Comment:
@Alexhuszagh does this not cover that? Runnign this on my machine this passes

Reply:
Oh right. I forgot to checkout the PR, because I did and forgot and had unstaged changes.